src/EnergyPlus/AirflowNetwork/CMakeLists.txt

```diff
@@ -10,7 +6,7 @@ set(SRCS src/Solver.cpp src/Elements.cpp src/Properties.cpp)
 # this will be linked statically to create the DLL and also the unit tests
 add_library(airflownetworklib STATIC ${SRCS} ${HDRS})
 
-if(CMAKE_COMPILER_IS_GNUCXX OR "${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang" OR "${CMAKE_CXX_COMPILER_ID}" STREQUAL "AppleClang") # g++/Clang
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID STREQUAL "Clang" OR CMAKE_CXX_COMPILER_ID STREQUAL "AppleClang") # g++/Clang
```

Comment:
With CMP0054 set to either OLD or NEW this would work the same so it should be the preferred way.

Reply:
:+1: